components/speedreader/rust/src/src/readability/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I think we should somehow add this to `brave://credits`, probably there is some related process?